lib/sdf-server/src/service/v2/func/binding/update_binding.rs

```diff
@@ -188,9 +158,6 @@ pub async fn update_binding(
         FuncKind::CodeGeneration | FuncKind::Qualification => {
             update_leaf_func_bindings(&ctx, request.bindings).await?;
         }
-        FuncKind::Management => {
```

Comment:
Updates should not be happening anymore, as there is nothing updateable in a management binding.